html/js-random.html

```diff
@@ -0,0 +1,15 @@
+<html>
+
+<body>
+    <button onclick="document.getElementById('demo').innerHTML = getRndInteger(0,100)">Click Me</button>
+
+    <p id="demo"></p>
+
+    <script>
+        function getRndInteger(min, max) {
```

Comment:
What is Rnd ?

Reply:
stands for random

Reply:
to be direct in practice we don't really do these short namings because we have autocomplete and readability is more important.